### PR TITLE
Better heartbeat for ProgressPrinter. See Issue #1928

### PR DESCRIPTION
**Pull request by vignesh-m**
Somewhat quickfix solution outlined in Issue #1928

### CODE REVIEW

File: bindings/python/cntk/logging/progress_print.py [hunk 1]
```diff
@@ -398,6 +398,7 @@ def on_write_training_summary(self, samples, updates, summaries, aggregate_loss,
         elapsed_seconds = elapsed_milliseconds / 1000
         speed = _avg(samples, elapsed_seconds)
         avg_loss = _avg(aggregate_loss, samples)
+        self.epochs = summaries
```
Comment (eldakms) [inline bindings/python/cntk/logging/progress_print.py]: I suppose this won't work in case of checkpoint/crash/restore, but still better than 0

Comment (vignesh-m) [inline bindings/python/cntk/logging/progress_print.py]: Yeah, I meant this as a quick fix to give some sense of progress

**Comment by eldakms** [comment]
Please check "summaries" - it is of type string in some cases (make sure CNTKv2Python/Examples pass). Could you also please rebase on the latest master? Thank you!

**Comment by cha-zhang** [comment]
Philly no longer need heartbeat signal. Close this now.

**Pull request closed without merging**